src/Symfony/Component/Process/Exception/ProcessFailedException.php

```diff
@@ -28,10 +28,11 @@ public function __construct(Process $process)
             throw new InvalidArgumentException('Expected a failed process, but the given process was successful.');
         }
 
-        $error = sprintf('The command "%s" failed.'."\nExit Code: %s(%s)",
+        $error = sprintf('The command "%s" failed.'."\n\nExit Code: %s(%s)\n\nWorking directory: %s",
             $process->getCommandLine(),
             $process->getExitCode(),
-            $process->getExitCodeText()
+            $process->getExitCodeText(),
+            $process->getWorkingDirectory()
```

Comment:
The goal is to improve error information if a process fails because of the lack of user rights on the working directory.